### DISCUSSION TITLE
I activated my Github Enterprise coupon, but I don't have any subscription.

### DISCUSSION THREAD

**Discussion by DamienDoumer** [community/community#121213] [New to GitHub]
I integrated the Microsoft for startup program and I was admitted and was offered Github copilot credit. But when I activate Github copilot, I can't see my subscription. Github tells me I've used my coupon, but I have no subscription.

**Comment by hayad7** [comment]
Did you find a solution ? I am facing a similar issue [is Copilot not available with the Github enterprise for startups coupon ?](https://github.com/orgs/community/discussions/121367)

**Reply by DamienDoumer** [reply]
Unfortunately, I didn't.

**Reply by heias** [reply]
Hi @DamienDoumer and @hayad7 , I have the same issue. Do you have a solution?

**Reply by DamienDoumer** [reply]
I waited a while, and the subscription was activated when I came back. I didn't do anything special in the end.

**Reply by rebecka-zavaleta-paismo** [reply]
I have the same issue.

**Comment by PRITHIVSAKTHIUR** [comment]
Reach out to GitHub support directly. They can investigate the issue and provide you with information on why you're unable to see your subscription despite activating the credit. You can usually find a support contact form or email address on the GitHub website.
🔗https://support.github.com/

**Comment by PRITHIVSAKTHIUR** [comment]
> I integrated the Microsoft for startup program and I was admitted and was offered Github copilot credit. But when I activate Github copilot, I can't see my subscription. Github tells me I've used my coupon, but I have no subscription.

https://support.github.com/
